website/content/docs/enterprise/automated-upgrades.mdx

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+page_title: Vault Enterprise Automated Upgrades
+description: |-
+  Vault Enterprise can upgrade itself automatically.
+---
+
+# Automated Upgrades
+
+-> **Note**: This feature requires [Vault Enterprise](https://www.hashicorp.com/products/vault/)
+configured to use Integrated Storage.
+
+Vault Enterprise Automated Upgrades allows operators to upgrade the version of Vault running in
+a cluster in an automated fashion. There are a few different ways to make this upgrade happen,
+and control which versions are being upgraded to. By default, with no additional configuration,
+Vault will check the version of Vault that each node in the cluster is running. If a blue/green
+style deploy is desired, Vault can be made to use a version of your choosing, regardless of
+what version of Vault is currently running.
+
+## Configuration
+A new key can be added to Vault's `storage` configuration stanza: `autopilot_upgrade_version`.
+The value for this key is a [SemVer](https://semver.org/) compatible version string of your choosing.
+If this is present, it will override the actual version of Vault that is running for the purposes of
+automated upgrades.
+
+## Mechanics
+Regardless of whether you choose to use Vault's built-in version, or a version of your choosing, the
+mechanics for performing an automated upgrade are the same. Once you have a Vault cluster running,
```

Comment:
```suggestion
mechanics for performing an automatic upgrades remain the same. Once you have a Vault cluster running,
```

Reply:
Under the Mechanics section, this is my suggested rewrite. Please feel fee to ignore if the rewrite changed the intent of this paragraph in any way.

**Suggested changes**: 
Whether you choose to use Vault's built-in version or a version of your own, the mechanics for performing automatic upgrades remain the same.

When a Vault cluster is running and new nodes containing an updated Vault version join the cluster, the Autopilot subsystem within Vault will promote the new version nodes to voters when the number of nodes running the latest Vault version equals or exceeds the number of pre-existing nodes. Leadership transfers from the prior leader to a randomly selected node running the newest Vault version and waits for the user to remove the previous nodes from the cluster. Vault then demotes the previous version's nodes to non-voters.

Reply:
It does change it slightly, but I like a lot of your wording better than mine. I will use your paragraph as much as possible, with a few slight tweaks.